source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cpp

```diff
@@ -1,6 +1,7 @@
 /*
 This example shows how to capture point clouds, with color, from the Zivid camera,
-with settings from YML file.
+with settings from YML file. The YML files for this samples can be found at:
+https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/450363393/Sample+Data
```

Comment:
We should avoid hard coding these kinds of instructions in the code samples. This is already covered in README.md. The reason to avoid doing this: if we change the download link, we have to change it in every sample that has it. We want to have only one place where we change it ideally.

If you want to emphasize, do it by saying that this is covered in the instructions.

![image](https://user-images.githubusercontent.com/46567886/87552362-df888a80-c6b1-11ea-97b4-6f83b1bc4f72.png)

Reply:
Updated with link removal.